scaling_performance/optimizing_storage.adoc

```diff
@@ -11,19 +11,167 @@
+must support RWX access mode and must ensure read-after-write consistency. File storage and block storage are expressly not recommended
+for a scaled/HA {product-title} registry cluster deployment with production workloads. All NAS storage back-end types
+(excluding CNS/CRS GlusterFS as it uses an object storage interface) are expressly not recommended for {product-title} Registry cluster deployment with production workloads.
+
+[WARNING]
+====
+Corruption may occur when using NFS to back {production-title} scaled/HA registry at production workloads.
+====
+
+[[metrics]]
+===== Metrics
+
+In {product-title} hosted metrics cluster deployment, the preferred storage back-end type is block storage. All NAS storage back-end type (excluding
+CNS/CRS GlusterFS as it uses a block storage interface via iSCSI) are expressly not recommended for hosted metrics cluster deployment with production workloads.
+
+[WARNING]
+====
+Corruption may occur when using NFS to back hosted metrics at production workloads.
+====
+
+[[logging]]
+===== Logging
+
+In {product-title} hosted logging cluster deployment, the preferred storage back-end type is block storage. All NAS storage back-end type (excluding CNS/CRS GlusterFS as it
+uses a block storage interface via iSCSI) are expressly not recommended for hosted logging cluster deployment with production workloads.
+
+[WARNING]
+====
+Corruption may occur when using NFS to back Hosted Logging at production workloads.
+====
+
+[[applications]]
+===== Applications
+
+Application use cases vary from application to application. Storage back-end types that support dynamic PV provisioning have low mount time latencies, and are not tied
+to nodes to support a healthy cluster. NFS does not guarantee read-after-write consistency and is not recommended for applications which require it. Applications
+which depend on writing to the same, shared NFS export may experience issues at production workloads.
+
+[[other-storage-recommendations]]
+==== Other Specific Application Storage Recommendations
+
+* {product-title} Internal etcd: For the best etcd reliability, the lowest consistent latency storage back-end type is preferable.
+* OpenStack Cinder: OpenStack Cinder tends to be adept in ROX access mode use cases.
+* Databases: Databases (RDBMSs, NoSQL DBs, etc.) tend to perform best with dedicated block storage.
+
+[[choosing-a-docker-graph-driver]]
+=== Choosing a Docker Graph Driver
 
 Docker stores images and containers in a graph driver (a pluggable storage
-backend), such as Device Mapper, Overlay, and Btrfs. Each have advantages and
-disadvantages. For example, Overlay is faster than Device Mapper at starting and
-stopping containers, but is not POSIX compliant because of the architectural
+back end), such as device mapper, overlay, and btrfs. Each have advantages and
+disadvantages. For example, overlay is faster than device mapper at starting and
```

Comment:
Shouldn't 'overlay' and 'device mapper' be styled as literals?

Reply:
:+1:

Reply:
Styled as `Overlay` and `DeviceMapper`.